ios/keyman/Keyman/Keyman/MainViewController.swift

```diff
@@ -132,7 +132,11 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
     Manager.shared.canRemoveDefaultKeyboard = true
 
     // Pre-load for use in update checks.
-    Manager.shared.apiKeyboardRepository.fetch()
+    if !ResourceDownloadManager.shared.updateCacheIsCurrent {
+      // TODO:  Actually use the query's results once available.
+      //        That said, this is as far as older versions used the query here.
+      ResourceDownloadManager.shared.queryKeysForUpdatablePackages { _, _ in }
+    }
```

Comment:
Additional follow-up work at this point in the code would allow update notifications like those we currently support in Keyman for Android - the (currently-empty) completion block is designed to facilitate this.

But, for now, this is enough to maintain iOS's "status quo" regarding resource updates.